packages/react-native-web/src/exports/Image/types.js

```diff
@@ -102,8 +102,8 @@ export type ImageStyle = {
   tintColor?: ColorValue
 };
 
-export type ImageProps = {
-  ...ViewProps,
+export type ImageProps = {|
+  ...$Exact<ViewProps>,
```

Comment:
I had `Flow` errors after my changes
Following the error messages and suggested fixes led to these changes